sirepo/uri.py

```diff
@@ -28,17 +22,21 @@
 PATH_INFO_CHAR = '*'
 
 
-def app_root(sim_type):
+def app_root(sim_type, external=False):
     """Generate uri for application root
 
     Args:
         sim_type (str): application name
+        external (bool): external uri? [False]
```

Comment:
`if True, make the uri absolute`

Try not to repeat the name of the parameter in the description to help reader understand. The `?` is not appropriate, because the client wants to know what the function will do when supplying it True or False, that is, write it from the perspective of the function, not what the client is asking.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c